wikibase/1.30/bundle/Dockerfile

```diff
@@ -1,12 +1,18 @@
 FROM ubuntu as fetcher
 
 RUN apt-get update && \
-    apt-get install --yes --no-install-recommends unzip jq curl ca-certificates
+    apt-get install --yes --no-install-recommends unzip jq curl ca-certificates git
```

Comment:
If you want to avoid an extra package, you could also download a tarball instead:

```sh
curl -sL https://github.com/filbertkm/WikibaseImport/archive/master.tar.gz | tar xz
```

(Note that the root directory in the archive is called `WikibaseImport-master`, not just `WikibaseImport`, so I assume you’d have to rename it at some point.)

Reply:
Excellent; I'll do that